docs/usage/configuration-options.md

```diff
@@ -642,24 +642,6 @@ Descriptions fields embedded within presets are also collated as part of the onb
 
 Add to this object if you wish to define rules that apply only to PRs that update digests.
 
-## docker
-
-Add config here if you wish it to apply to Docker package managers Dockerfile and Docker Compose.
-If instead you mean to apply settings to any package manager that updates using the Docker _datasource_, use a package rule instead, e.g.
-
-```json
-{
-  "packageRules": [
-    {
-      "matchDatasources": ["docker"],
-      "labels": ["docker-update"]
-    }
-  ]
-}
-```
-
-## dotnet
```

Comment:
If someone today configures:

```
{
  "dotnet": {
    "enabled": false
  }
}
```

Then it would completely disable the `nuget` manager from even trying to extract.

With this change, users will see the `nuget` manager matching/extracting but then anything which is extracted is then disabled?

Reply:
The validation would fail as categories are not allowed as top level objects.

Reply:
Wouldn't it migrate?

Second, I think we still support using manager and datasource names as top level objects, don't we? Some of these languages like `docker` and `dotnet` could therefore be valid at the top level.

Reply:
Currently not. 

Yes, manager are supported on the top level, but I'm not aware that the we support datasources there. 🤔      
At least that is how interpret the [validation code](https://github.com/renovatebot/renovate/pull/16534/files#diff-aa68cda516aaa0c65459a20c907086b0e6b7e8a930a429f70ca5ed46cdcd5676L27)
Further there is no overlap between languages and managers.

Reply:
> Do I understand right that you're retaining the option to configure top-level manager objects, but not languages?

Yes, that's correct.